src/installer.ts

```diff
@@ -241,9 +240,28 @@ export class DotnetCoreInstaller {
     return downloadUrls;
   }
 
-  private getReleasesJson(): Promise<HttpClientResponse> {
-    var httpCallbackClient = new httpClient.HttpClient('setup-dotnet', [], {});
-    return httpCallbackClient.get(DotNetCoreReleasesUrl);
+  private async getReleasesJsonUrl(
+    httpCallbackClient: httpClient.HttpClient,
+    versionParts: string[]
+  ): Promise<string> {
+    // First, get the location of the correct releases.json then get that.
```

Comment:
"then get that"? Is there something missing from end of sentence?

Reply:
I think that got left when I refactored. We don't need it, doesn't add anything to the function name.